orchagent/portsorch.cpp

```diff
@@ -2098,6 +2099,11 @@ bool PortsOrch::initializePort(Port &p)
         FieldValueTuple tuple("oper_status", "down");
         vector.push_back(tuple);
         m_portTable->set(p.m_alias, vector);
+        p.m_oper_status = SAI_PORT_OPER_STATUS_DOWN;
+    }
+    else
+    {
+        p.m_oper_status = SAI_PORT_OPER_STATUS_UP;
```

Comment:
Does this match what the db has? Any concerns about the possibility of a mismatch?

Reply:
Yes,  operStatus is the value read from appDB.

https://github.com/Azure/sonic-swss/blob/8bfdea086f14bbe420846d42c607760999cbdcc3/orchagent/portsorch.cpp#L2093
p.m_oper_status is a mirror of what set in DB.